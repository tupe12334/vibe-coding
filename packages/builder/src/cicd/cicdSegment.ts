import type { RootContent } from "mdast";

export const cicdSegment = async (system: string): Promise<RootContent[]> => {
  const segment: RootContent[] = [
    {
      type: "paragraph",
      children: [
        {
          type: "text",
          value: `CI/CD system: ${system}`,
        },
      ],
    },
  ];

  return segment;
};
