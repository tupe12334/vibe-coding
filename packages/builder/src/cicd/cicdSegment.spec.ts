import { describe, expect, test } from "vitest";
import { cicdSegment } from "./cicdSegment";

describe("cicdSegment", () => {
  test("returns cicd system line", async () => {
    const segment = await cicdSegment("github-actions");
    expect(segment).toMatchSnapshot();
  });
});
