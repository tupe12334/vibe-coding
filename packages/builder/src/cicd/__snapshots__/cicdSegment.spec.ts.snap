// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`cicdSegment > returns cicd system line 1`] = `
[
  {
    "children": [
      {
        "type": "text",
        "value": "CI/CD system: github-actions",
      },
    ],
    "type": "paragraph",
  },
]
`;
